Azure/azure-cli
# [KeyVault] Add new parameter `--enable-rbac-authorization` during creating or updating

Support for #11268 

- [x] Waiting for Swagger/Python SDK, release time: Apr 15.
- [x] Waiting for service public.

How to test:

1. Run `az keyvault create -n {} -g {} --enable-rbac-authorization`
2. Check the value of `properties.enableRbacAuthorization`

---

This checklist is used to make sure that common guidelines for a pull request are followed.
